x-pack/legacy/plugins/siem/public/components/charts/common.tsx

```diff
@@ -17,14 +17,18 @@ import {
   ScaleType,
 } from '@elastic/charts';
 import { i18n } from '@kbn/i18n';
-import { TickFormatter } from '@elastic/charts/dist/lib/series/specs';
+import { TickFormatter, Rotation, Rendering } from '@elastic/charts/dist/lib/series/specs';
 import chrome from 'ui/chrome';
 import moment from 'moment-timezone';
+import { SettingSpecProps } from '@elastic/charts/dist/specs/settings';
 
 const chartHeight = 74;
+export const chartDefaultRotation: Rotation = 0;
```

Comment:
Does this need to be exported? I don't see any usages outside this file currently.

Reply:
Right, let me remove that, Thanks @spong ! You've got eagle eyes!
![eagle eyes](https://media.giphy.com/media/RKw94AILIkuWY/giphy.gif)